server/api/googleLogin.js

```diff
@@ -80,15 +65,15 @@ async function googleSignup(req, res, username) {
  * @param {Database} db - The database to insert the User into.
  * @param {String} username - The username of the new User.
  * @param {Callback} callback - A callback which handles what happens after inserting the user.
- *                   Should take in two arguments: an error `error` and the `user` object generated from `insertUser`.
- *                   `user` will be an empty object if an error is thrown, and `error` will be null if there is no error.
+ *                   Should take in one argument: an error `error`
+ *                   `error` will be null if there is no error.
  * @returns Nothing.
  */
 async function insertUser(db, username, callback) {
```

Comment:
Perhaps this should be renamed as it is functionally different from `signup.js`'s `insertUser`? (perhaps to `insertGoogleUser`)?

I know this probably shouldn't be work for this user story but I don't know where else we'd squeak it in

Reply:
i'd say it can work here since it's "code cleanup" in an area i'm already messing with